docs/user/FAQS.md

```diff
@@ -27,19 +27,22 @@ git pull
 Now your local copy of ORFS should be up-to-date. The next step is to build it again with the following command: 
 
 For WSL/docker based installation, run:
-```
+```shell
 ./build_openroad.sh --clean
 ```
 
 For local installation, run:
-```
+```shell
 ./build_openroad.sh --local --clean
 ```
 
 For pre-built binaries, you can run the flow after `git pull` is done.
 
-> **TIP**: For development purposes, it is a good practice to work on branches and leave master untouched. 
-> To update your branch, simply run `git checkout <BRANCH>; git merge master`
+:::{tip} 
+For development purposes, it is a good practice to work on branches and leave master untouched. 
+To update your branch, simply run `git checkout <BRANCH>; git merge master`
```

Comment:
```suggestion
To update your branch, simply run `git checkout -b <BRANCH> && git merge master`
```

Reply:
@vvbandeira Won't git checkout -b return an error if branch exists?

Reply:
Good point, but calling `checkout` on a branch that does not exists also leads to an error. Please add a note for existing and first-time.

Reply:
@vvbandeira You reckon we should add a blurb on adding forks / adding upstream links etc for contributor guide?